client/src/pages/Courses/editables/SaveCancelButtonSet.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Flex, Button } from "@chakra-ui/core";
+import { colors as c } from "../../../colors";
+
+const SaveCancelButtonSet = ({ onCancel, onSave, isActive }) => {
```

Comment:
Do you think we should refactor this into the `components` folder instead of `Courses` so we can reuse it later on? We should also style it to be consistent in a future PR

Reply:
Yup we can refactor it in a different task